src/extension.ts

```diff
@@ -1,51 +1,114 @@
-/* --------------------------------------------------------------------------------------------
- * Copyright (c) Microsoft Corporation. All rights reserved.
- * Licensed under the MIT License. See License.txt in the project root for license information.
- * ------------------------------------------------------------------------------------------ */
-'use strict';
-
-import * as net from 'net';
-
-import { workspace, Disposable, ExtensionContext } from 'vscode';
-import { LanguageClient, LanguageClientOptions, SettingMonitor, ServerOptions, ErrorAction, ErrorHandler, CloseAction, TransportKind } from 'vscode-languageclient';
-
-function startLangServer(command: string, documentSelector: string | string[]): Disposable {
-	const serverOptions: ServerOptions = {
-		command: command,
-	};
-	const clientOptions: LanguageClientOptions = {
-		documentSelector: documentSelector,
-	}
-	return new LanguageClient(command, serverOptions, clientOptions).start();
+"use strict";
+
+import * as net from "net";
+import { resolve, } from "path";
+
+import {
+    Disposable,
+    ExtensionContext,
+    workspace,
+    WorkspaceConfiguration
+} from "vscode";
+import {
+    CloseAction,
+    ErrorAction,
+    ErrorHandler,
+    LanguageClient,
+    LanguageClientOptions,
+    ServerOptions,
+    SettingMonitor,
+    TransportKind,
+} from "vscode-languageclient";
+
+const coalals = {
+    langs: [
+        "html",
+        "python",
+    ],
+
+    command: {
+        usage: resolve(__dirname, "../coalals/coala-ls.sh"),
```

Comment:
Can we avoid a shell script?  Not always gonna work on Windows, the primary audience for this plugin